packages/main/src/plugin/kubernetes-context-state.ts

```diff
@@ -547,6 +676,19 @@ export class ContextsManager {
   }
 
   public getCurrentContextResources(resourceName: ResourceName): KubernetesObject[] {
-    return this.states.getCurrentContextResources(this.kubeConfig.currentContext, resourceName);
+    if (!this.currentContext) {
+      return [];
+    }
+    if (this.states.hasInformer(this.currentContext, resourceName)) {
+      console.debug(`informer for ${resourceName} in ${this.currentContext} running`);
```

Comment:
as said previously I think that informer is for the programmer but as a end-user I would expect more what it does like

`Already watching ${resourceName} of ${context}`